management/virtualmachine/client.go

```diff
@@ -461,27 +460,24 @@ func (self VirtualMachineClient) ResolveRoleSize(roleSizeName string) error {
 }
 
 func (self VirtualMachineClient) createStartRoleOperation() StartRoleOperation {
-	startRoleOperation := StartRoleOperation{}
-	startRoleOperation.OperationType = "StartRoleOperation"
-	startRoleOperation.Xmlns = azureXmlns
-
-	return startRoleOperation
+	return StartRoleOperation{
+		OperationType: "StartRoleOperation",
+		Xmlns:         azureXmlns,
+	}
 }
 
 func (self VirtualMachineClient) createShutdowRoleOperation() ShutdownRoleOperation {
-	shutdownRoleOperation := ShutdownRoleOperation{}
-	shutdownRoleOperation.OperationType = "ShutdownRoleOperation"
-	shutdownRoleOperation.Xmlns = azureXmlns
-
-	return shutdownRoleOperation
+	return ShutdownRoleOperation{
+		OperationType: "ShutdownRoleOperation",
+		Xmlns:         azureXmlns,
```

Comment:
are we testing this manually by any chance? Am I safe to merge it? :-) 


On Monday, Mar 16, 2015 at 3:49 PM, Paul Meyer <notifications@github.com>, wrote:

In management/virtualmachine/client.go:

>  }
>  
>  func (self VirtualMachineClient) createShutdowRoleOperation() ShutdownRoleOperation {
> -	shutdownRoleOperation := ShutdownRoleOperation{}
> -	shutdownRoleOperation.OperationType = "ShutdownRoleOperation"
> -	shutdownRoleOperation.Xmlns = azureXmlns
> -
> -	return shutdownRoleOperation
> +	return ShutdownRoleOperation{
> +		OperationType: "ShutdownRoleOperation",
> +		Xmlns:         azureXmlns,

I don't think so. I'll remove all of them...


—
Reply to this email directly or view it on GitHub.